openshift/ups-template.json

```diff
@@ -0,0 +1,529 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "redhat-mobile-backend",
+    "annotations": {
+      "description": "An example Mobile Backend using RedHat, JBoss, AeroGear, and FeedHenry technologies",
```

Comment:
Yeah, this code is part of something else I am working on, the UPS integration was just the first piece I got done.  It just so happened I did it when a bunch of people on aerogear-dev were asking about it.